packages/dpcpp/package.py

```diff
@@ -46,10 +46,12 @@ class Dpcpp(Package):
     # TODO: Figure out how to get those versions
     # programmatically from the intel-oneapi-compilers package?
     available_versions = [
+        "2022.2.1",
         "2022.2.0",
         "2022.1.0",
         "2022.0.2",
         "2022.0.1",
+        "2021.7.1",
```

Comment:
We actually shouldn't put this in here. It is only the classic (legacy) Intel compilers that are provided at this version. There is no version 2021.7.1 for OneAPI. As DPCPP doesn't work with the legacy compilers, there is no point listing a release of them.